libtenzir/builtins/contexts/lookup_table.cpp

```diff
@@ -324,8 +358,14 @@ class ctx final : public virtual context {
     while (key_it != key_values.end()) {
       TENZIR_ASSERT(context_it != context_values.end());
       auto materialized_key = materialize(*key_it);
-      context_entries.insert_or_assign(materialized_key,
-                                       materialize(*context_it));
+      // Subnets never make it into the regular map of entries.
```

Comment:
I think this makes everything a lot more complicated than it needs to be. You can remove a lot of the code in this file if you just add it to both.

Reply:
But that 2x'es the space utilization.

Reply:
I think this is irrelevant until someone complains about it. Let's wait with improving this until someone wants to add a few million subnets.

Reply:
Okay. YOLO but fine with me for the first version.

Reply:
Even then, if you really want to optimize it, you could still keep it separate in memory but persist it together as one lookup table. No need to change the on-disk format either way.